tests/Mapper/RecordTest.php

```diff
@@ -118,4 +118,22 @@ public function testHas()
         // missing
         $this->assertFalse($this->record->has('noSuchField'));
     }
+
+    public function testSet()
+    {
+        $this->record->set([
+            'foo' => 'hello',
+            'zim' => 'dim'
+        ]);
+
+        $actual = $this->record->getArrayCopy();
```

Comment:
The getArrayCopy is failing. 

Didn't understand why so. May be late night :) .

```
There was 1 error:

1) Atlas\Orm\Mapper\RecordTest::testSet
Error: Call to a member function getArrayCopy() on string

/var/www/projects/github.com/harikt/Atlas.Orm/src/Mapper/Related.php:147
/var/www/projects/github.com/harikt/Atlas.Orm/src/Mapper/Record.php:211
/var/www/projects/github.com/harikt/Atlas.Orm/tests/Mapper/RecordTest.php:129
```